db/migrations/00022_add_description_and_tracker_id_in_task.go

```diff
@@ -0,0 +1,44 @@
+package migrations
+
+import (
+	"database/sql"
+	"github.com/jinzhu/gorm"
+	"github.com/pressly/goose"
+)
+
+func init() {
+	goose.AddMigration(Up00022, Down00022)
+}
+
+type task struct {
```

Comment:
We can only declare types once per package, so move it inside the methods below to keep it consistent.

Reply:
done